_studio/hevce_hw/CMakeLists.txt

```diff
@@ -47,8 +47,9 @@ set( sources.plus "" )
 set( LIBS "" )
 set( defs "" )
 
-include_directories( ${MSDK_STUDIO_ROOT}/hevce_hw/include )
-include_directories( ${MSDK_STUDIO_ROOT}/hevce_hw/h265/include )
+include_directories( "${MSDK_STUDIO_ROOT}/hevce_hw/h265/include" )
```

Comment:
why you need this in this? I think you need drop it.

Reply:
There are no folder hwce_hw/include, but there is hevce_hw/h265/include.